Makefile

```diff
@@ -46,7 +46,7 @@ CUDA_DIR  ?= $(or $(patsubst %/,%,$(dir $(patsubst %/,%,$(dir \
 # export LSAN_OPTIONS=suppressions=.asanignore
 AFLAGS = -fsanitize=address #-fsanitize=undefined -fno-omit-frame-pointer
 
-OPT    = -O -g
+OPT    = -O3 -g
```

Comment:
Please let's not change this in the global makefile. The README.md already encourages users to use suitable target/optimization flags.

Reply:
Also, why is `MAGMA_DIR`'s default a different schema from all the other packages and why is `NVCC` not respecting `CUDA_DIR`?

Reply:
This version of the the makefile in the magma-dev branch is not up to date with the one under master. For example, the -O flag is originally rejected by nvcc, but I can see that this issue has already been fixed in the master version (using Xcompiler). The same applies, I believe, for MAGMA_DIR and CUDA_DIR. I will undo this edit, but a proper merge is required.